pkg/okteto/context.go

```diff
@@ -191,7 +191,10 @@ func K8sContextToOktetoUrl(ctx context.Context, k8sContext, k8sNamespace string,
 
 	// check the namespace label
 	if k8sNamespace == "" {
-		k8sNamespace = cfg.Contexts[k8sContext].Namespace
+		cfgK8sNamespace, exists := cfg.Contexts[k8sContext]
```

Comment:
This code was throwing a panic. I though about refactoring it a bit by creating a struct and not using global variables. Let me know if you don't think it should be addressed in this PR but since this PR uses this function several times I think is a good refactor to do it

Reply:
I think is worth changing it here